[2024-12-23T19:59:16.850+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-23T19:59:16.880+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_connect.bash_ex1 scheduled__2024-12-20T08:00:00+00:00 [queued]>
[2024-12-23T19:59:16.891+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_connect.bash_ex1 scheduled__2024-12-20T08:00:00+00:00 [queued]>
[2024-12-23T19:59:16.891+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-23T19:59:16.911+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): bash_ex1> on 2024-12-20 08:00:00+00:00
[2024-12-23T19:59:16.919+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1843) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-23T19:59:16.920+0000] {standard_task_runner.py:63} INFO - Started process 1847 to run task
[2024-12-23T19:59:16.921+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_connect', 'bash_ex1', 'scheduled__2024-12-20T08:00:00+00:00', '--job-id', '1053', '--raw', '--subdir', 'DAGS_FOLDER/dag_connect.py', '--cfg-path', '/tmp/tmp_ur43f5g']
[2024-12-23T19:59:16.925+0000] {standard_task_runner.py:91} INFO - Job 1053: Subtask bash_ex1
[2024-12-23T19:59:16.996+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_connect.bash_ex1 scheduled__2024-12-20T08:00:00+00:00 [running]> on host 87a1e11545b0
[2024-12-23T19:59:17.127+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_connect' AIRFLOW_CTX_TASK_ID='bash_ex1' AIRFLOW_CTX_EXECUTION_DATE='2024-12-20T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-20T08:00:00+00:00'
[2024-12-23T19:59:17.130+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-23T19:59:17.152+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-23T19:59:17.153+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Task 1"']
[2024-12-23T19:59:17.172+0000] {subprocess.py:86} INFO - Output:
[2024-12-23T19:59:17.176+0000] {subprocess.py:93} INFO - Task 1
[2024-12-23T19:59:17.180+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-23T19:59:17.181+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-23T19:59:17.215+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_connect, task_id=bash_ex1, run_id=scheduled__2024-12-20T08:00:00+00:00, execution_date=20241220T080000, start_date=20241223T195916, end_date=20241223T195917
[2024-12-23T19:59:17.258+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-23T19:59:17.278+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-23T19:59:17.280+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
